modules/price/src/CurrencyMismatchException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Drupal\commerce_price;
+
+/**
+ * This exception is thrown when attempting to perform math using two different currencies.
+ * For example, adding a price in USD to another in EUR.
+ */
+class CurrencyMismatchException extends \RuntimeException {
```

Comment:
That's an easy change so let me know if this is something you'd like implemented.